.gitignore

```diff
@@ -14,4 +14,7 @@ node_modules/
 # Compiled production code
 /dist/
 .yalc
-yalc.lock
\ No newline at end of file
+yalc.lock
+
+# Databases
+database*
```

Comment:
you can ignore my comment in previous PRs

Why not `database/`

Reply:
There may be different database directories. We should ignore them all :slightly_smiling_face: